File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.java
class Solution {
    public String solution(String s) {
        String answer = "";
        String tmp[] = s.split(" ");
        int min ,max, n;
        n = Integer.parseInt(tmp[0]);
        min = n;
        max = n;
        for(var i=0;i<tmp.length;i++)
        {
            n = Integer.parseInt(tmp[i]);
            if(min > n) min = n;
            if(max < n) max = n;
        }
        return min + " " + max;
    }
}

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.cpp
#include <string>
#include <vector>
#include <algorithm>
#include <sstream>

using namespace std;

string solution(string s) {
    string answer = "";
    stringstream ss(s);
    string tmp;
    vector<int> v;
    while(ss>>tmp){
        v.push_back(stoi(tmp));
    }
    sort(v.begin(),v.end());
    answer +=to_string(v[0]);
    answer +=" ";
    answer +=to_string(v[v.size()-1]);
    return answer;
}